src/modules/pos/validator/components/ChangeCommission/Status/index.js

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { selectActiveTokenAccount } from 'src/redux/selectors';
+import { transactionBroadcasted, resetTransactionResult } from 'src/redux/actions';
+import Status from './ChangeCommissionStatus';
+
+const mapStateToProps = state => ({
+  account: selectActiveTokenAccount(state),
+  transactions: state.transactions,
+});
+const mapDispatchToProps = {
```

Comment:
These action functions are not used in the `Status` component. Is this preparing for something else or can we remove them?
If it is a standard status component we could also use the generic 

`import TxBroadcasterWithStatus from '@transaction/components/TxBroadcasterWithStatus/TxBroadcasterWithStatus';`

inside the ChangeCommission.js <Multistep>.

To prevent code duplications.

Reply:
it should be part of the TxBroadcaster and its refactore dows think is not part of this ticket we can create an other ticket for it

Reply:
Yes, lets wait, might change a bit if we add error messages to our failed transactions